NiyongaboEric/table-topic-question-api
# #171915633 Check if user created profile

#### What does this PR do? 
The PR helps to check if the user created a profile and update the token


#### Description of Task to be completed?
**Expected**
- The profile is checked
- the environment variable is used to hide url
- Validating image message is refactored
- include profileCreated key-value pair in token
- URLs are in `.env file`
- find one query was added

#### How should this be manually tested?
- `git clone https://github.com/NiyongaboEric/table-topic-question-api.git`
- cd table-topic-question-api
- `git checkout ch-check-profile-refactoring-171915633`
- `npm install`
- login and check the token 

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#171915633](https://www.pivotaltracker.com/story/show/171915633)

#### Screenshots (if appropriate)
<img width="1439" alt="Screen Shot 2020-03-22 at 15 51 29" src="https://user-images.githubusercontent.com/33821734/77251106-1ba46500-6c55-11ea-909c-5e68dcd79ff8.png">


#### Questions:
N/A
